core/packages/test/README.md

```diff
@@ -6,50 +6,29 @@ The E2E tests run against local deployments of the parachain, relayer, the ether
 
 * Ubuntu 20.04 or later. MacOS may work, but it's not currently a supported configuration.
 * Development environment for Rust and Substrate. See parachain [requirements](../parachain/README.md#requirements).
-* Node 16. Can install using [nvm](https://github.com/nvm-sh/nvm#installing-and-updating):
-
-  ```bash
-  nvm install 16.17.0
-  nvm use 16.17.0
-  ```
```

Comment:
Its in `core/.nvmrc` :

```
vincent@oryx:core$ cat .nvmrc
v16.18.1
```

So devs just need to run `nvm use`, and it will pickup the version from the config.

Reply:
Awesome!